src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts

```diff
@@ -374,10 +373,12 @@ export class DebugEditorContribution implements IDebugEditorContribution {
 		const sf = this.debugService.getViewModel().focusedStackFrame;
 		const model = this.editor.getModel();
 		if (sf && model && this.uriIdentityService.extUri.isEqual(sf.source.uri, model.uri) && !this.altPressed) {
+			this.editor.updateOptions({ hover: { enabled: false } });
```

Comment:
I'm seeing a double hover, because the editor has already shown its hover before this runs 
<img width="296" alt="image" src="https://github.com/microsoft/vscode/assets/323878/aa913def-7113-461c-b5d0-91dff638a030">

Reply:
I see , so the debug hover is scheduled to run after a delay since the editor hover was previously enabled you see that again. 

Then would it make sense to disable editor hover before the schedule ?

May be before this [line](https://github.com/microsoft/vscode/blob/b1bb819bbf2f682ac5050203e51f545324111522/src/vs/workbench/contrib/debug/browser/debugEditorContribution.ts#L479)

Reply:
I think that might cause the same issue that you describe below? But, you can try it

Reply:
It looks good . You can check the latest changes.